eva-accession-clustering-automation/clustering_automation/create_clustering_progress_table.py

```diff
@@ -100,19 +102,62 @@ def fill_in_from_previous_inventory(private_config_xml_file, release_version):
             execute_query(pg_conn, query_insert)
 
 
+def fill_num_rs_id_for_taxonomy_and_assembly(mongo_source, private_config_xml_file):
```

Comment:
Would it be possible to limit the run of the script to a taxonomy id and retrieve the assembly accession from the EVAPRO ?

Reply:
done